### PR TITLE
[green][metric/2] compute number of blockers by team

### PR DESCRIPTION
**Pull request by can-anyscale**
Compute number of blockers by team.

Test:

```
bazel run //ci/ray_ci/automation:weekly_green_metric
[INFO 2024-01-22 22:27:06,659] credentials.py: 1124  Found credentials in environment variables.
[INFO 2024-01-22 22:27:07,663] weekly_green_metric.py: 38  Found 17 release blockers
[INFO 2024-01-22 22:27:12,573] weekly_green_metric.py: 43        - core: 10
[INFO 2024-01-22 22:27:12,575] weekly_green_metric.py: 43        - rllib: 6
[INFO 2024-01-22 22:27:12,575] weekly_green_metric.py: 43        - ml: 1
```

### CODE REVIEW

File: ci/ray_ci/automation/weekly_green_metric.py [hunk 1]
```diff
@@ -34,5 +50,13 @@ def _get_release_blockers(
     return github.get_user().get_issues(state="open", labels=[release_blocker_label])
 
 
+def _get_oncall(issue: github.Issue.Issue) -> str:
+    labels = issue.get_labels()
+    for label in labels:
+        for team in TEAM:
+            if label.name == team:
+                return team
```
Comment (aslonnie) [inline ci/ray_ci/automation/weekly_green_metric.py:58]: this is basically `if label.name in TEAM: return label.name` ?

add unit test for this function / these changes?

**Comment by can-anyscale** [comment]
Added unit tests

File: ci/ray_ci/automation/weekly_green_metric.py (hunk 1 above)
Comment (aslonnie) [reply ci/ray_ci/automation/weekly_green_metric.py]: > this is basically `if label.name in TEAM: return label.name` ?

no?

File: release/ray_release/test_automation/state_machine.py [hunk 2]
```diff
@@ -16,6 +16,16 @@
...
+NO_TEAM = "none"
+TEAM = [
+    "core",
+    "data",
+    "kuberay",
+    "ml",
+    "rllib",
+    "serve",
+    "serverless",
+]
```
Comment (aslonnie) [inline release/ray_release/test_automation/state_machine.py:28]: why does this team list need to be repeated in two different files? maybe get a `py_library` and import it?

Comment (can-anyscale) [reply release/ray_release/test_automation/state_machine.py:28]: oh the other one is duplicated

File: ci/ray_ci/automation/weekly_green_metric.py (hunk 1 above)
Comment (can-anyscale) [reply ci/ray_ci/automation/weekly_green_metric.py]: i moved the logic to `release/ray_release/test_automation/state_machine.py` so we can have some test cases

**Review by aslonnie — APPROVED** [review]

**Pull request merged**